src/components/App/App.js

```diff
@@ -1,14 +1,19 @@
 
 import './App.css';
-import { Link } from 'react-router-dom';
+import { Link, NavLink } from 'react-router-dom';
 import Home from '../Home/Home';
 
 function App() {
   return (
   <div>
-    <Link to='/'>
-      <h1>Horizon Hikes</h1>
-    </Link>
+    <nav>
+      <Link to='/'>
```

Comment:
Nice! So your header is also a link home?

Reply:
Yes!